scholia/app/templates/base.html

```diff
@@ -99,6 +99,12 @@
 	 }
 	 catch(e) {}
 
+	 try {
+	     var doi = item.claims.P356[0].mainsnak.datavalue.value;
+	     detailsList.push( '<meta name="citation_doi" content="' + doi + '"/>');
+	 }
+	 catch(e) {}
+
 	 try {
 	     $( '#details' ).append( detailsList.join( " | " ) );
```

Comment:
Why is this not added to the head instead, like https://stackoverflow.com/questions/11018638/using-jquery-to-add-metadata-in-the-head-tag ?

Reply:
`$("head").append(<meta name="citation_doi" content="' + doi + '"/>');` ?

Reply:
Wasn't needed for the plugin to pick it up. But sounds good to me. I did not know the `$("head")` functionality.

Reply:
done.